defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+__elasticsearch_version: "2.x"
```

Comment:
Also, can you add the two variables (renamed) into README.md for documentation purposes?